crates/oxc_codegen/src/lib.rs

```diff
@@ -72,6 +80,16 @@ pub struct Codegen<const MINIFY: bool> {
     indentation: u8,
 
     sourcemap_builder: Option<SourcemapBuilder>,
+    comment_gen_related: Option<CommentGenRelated>,
```

Comment:
Can we put it in `CodegenOptions`? Also, it doesn't look like this feature is used anywhere. Do we need this?

Reply:
IMO, `Options` should store configuration like `boolean` type, but `CommentGenRelated` store trivias, and `MoveMap`, I don't think it is a good idea to put it under `CodegenOptions`

Reply:
more details you could refer: https://github.com/rolldown/rolldown/issues/1269

Reply:
Ok, I got it. You should add some tests for it.

Reply:
https://github.com/oxc-project/oxc/pull/3465/files#diff-8b2f51c395bbba32ab8432d172a0c5a85ab39159eac6ec709d29d0b77b2625d3R207